specifyweb/frontend/js_src/lib/components/AppResources/Editor.tsx

```diff
@@ -213,7 +220,11 @@ export function AppResourceEditor({
   );
 
   return typeof resourceData === 'object' ? (
-    <Container.Base className="flex-1 overflow-auto sm:overflow-visible">
```

Comment:
What purpose did this serve previously? I'm trying to comprehend why we would retain it rather than checking the user preferences for determining the appropriate class to use. 

@maxpatiiuk ?

Reply:
When `sm:overflow-visible` is not applied issue #4212 does not occur

Reply:
Appears to have been added by @CarolineDenis in https://github.com/specify/specify7/commit/1078bccd1 as part of improving app resource layout on small screens (probably as part of https://github.com/specify/specify7/issues/2617)